addons/point_of_sale/tests/test_point_of_sale_flow.py

```diff
@@ -405,10 +405,12 @@ def test_order_to_picking02(self):
             'type': 'product',
             'tracking': 'lot',
             'available_in_pos': True,
+            'company_id': self.company_data['company'].id,
```

Comment:
There is a lot of places where you add this `company_id` but is it always necessary ?
Isn't it hiding real problems where the taxes are not filtered by company, or the test/code does operations with the wrong company(ies) in the environment ?

Reply:
Indeed, this is a bug in the ORM and you can read more about it odoo/odoo/pull/132795. Until this is fixed, we resort to the `invalidate_model()` function for multi-company tests, and for tests not requiring a multi-company setup we add the company_id.